pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -79,13 +79,22 @@
     rank_zero_warn,
 )
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.imports import _HOROVOD_AVAILABLE, _HPU_AVAILABLE, _IPU_AVAILABLE, _TPU_AVAILABLE
+from pytorch_lightning.utilities.imports import (
+    _HOROVOD_AVAILABLE,
+    _HPU_AVAILABLE,
+    _IPU_AVAILABLE,
+    _TORCH_GREATER_EQUAL_1_11,
+    _TPU_AVAILABLE,
+)
 
 log = logging.getLogger(__name__)
 
 if _HOROVOD_AVAILABLE:
     import horovod.torch as hvd
 
+if _TORCH_GREATER_EQUAL_1_11:
+    from pytorch_lightning.strategies import DDPFullyShardedNativeStrategy
```

Comment:
I think it's a better experience that the class can be imported without this check, and we simply raise an exception if torch is not updated on initialization